src/main/java/uk/gov/hmcts/sptribs/ciccase/model/ContactPartiesCIC.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.sptribs.ciccase.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import uk.gov.hmcts.ccd.sdk.api.HasLabel;
+
+@Getter
+@AllArgsConstructor
+public enum ContactPartiesCIC implements HasLabel {
+
+    @JsonProperty("SubjectToContact")
+    SUBJECTTOCONTACT("Subject"),
+
+    @JsonProperty("RepresentativeToContact")
+    REPRESENTATIVETOCONTACT("Representative"),
+
+    @JsonProperty("ApplicantToContact")
+    APPLICANTTOCONTACT("Applicant (if different from subject)");
+
+    private final String label;
+
+    public boolean isSubject() {
+        return SUBJECTTOCONTACT.name().equalsIgnoreCase(this.name());
```

Comment:
For enum equality just use == ?

Reply:
Ok. but at the moment  we do not need it ,  will change it in next ticket.

Reply:
Please do, it's a smell.